README.md

```diff
@@ -154,10 +156,17 @@ using 8 parallel threads and writing the mapping output to `/dev/null`.
 
 To partition the index to obtain a meta-colored Fulgor index, then do:
 
-	./fulgor partition -i ~/Salmonella_enterica/salmonella_4546.fur -d tmp_dir --check
+	./fulgor meta -i ~/Salmonella_enterica/salmonella_4546.fur -d tmp_dir --check
+We can change the first argument from `meta` to `differential` or `meta-differential` to create a differential-colored or a meta-differential-colored index, respectively.
+
+See the table below for some additional data on the different indexes
+
+| command             | output file             | size (GB) | compression factor |
+|:--------------------|:------------------------|-----------|--------------------|
+| `meta`              | `salmonella_4546.mfur`  | 0.11769   | 2.26               |
+| `differential`      | `salmonella_4546.dfur`  | 0.11076   | 2.40               |
+| `meta-differential` | `salmonella_4546.mdfur` | 0.11076   | 2.84               |
```

Comment:
The size in GB here matches between `differential` and `meta-differential`, while the compression factors are different. Is this a copy-paste bug?

Reply:
Yes, it was an error. Thanks for pointing it out!